commands/databases.go

```diff
@@ -511,13 +513,57 @@ func RunDatabaseResize(c *CmdConfig) error {
 	}
 
 	id := c.Args[0]
+	dbs := c.Databases()
 
 	r, err := buildDatabaseResizeRequestFromArgs(c)
 	if err != nil {
 		return err
 	}
 
-	return c.Databases().Resize(id, r)
+	// Resize the database
+	err = dbs.Resize(id, r)
+	if err != nil {
+		return err
+	}
+
+	// Retrieve the database object after resize (to check status or further actions)
+	db, err := dbs.Get(id)
+	if err != nil {
+		return fmt.Errorf("failed to retrieve the database after resize: %v", err)
+	}
```

Comment:
I think that we could probably simplify this a bit further. Compared to the create command, the resize command only returns success or error. It doesn't output the database information. So we can remove this call to `Get` here as well as the one below in the `wait` block.